<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/bessieIcon.ico</normaloff>:/images/bessieIcon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="MechanicalDataGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string>Mechanical Data</string>
    </property>
    <widget class="QLabel" name="BrakePedalImage">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>520</y>
       <width>101</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>Brake Pedal Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="AccelPedalImage">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>520</y>
       <width>111</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>Accelerator Pedal Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="BrakePedalLbl">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>710</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brake</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AccelPedalLbl">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>710</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Accelerator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="BrakePedalProgressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>680</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>13</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="AccelPedalProgressBar">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>680</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>66</number>
     </property>
    </widget>
    <widget class="QLabel" name="SpeedLbl">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed (mph)</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="CurrentSpeedLCD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>381</width>
       <height>251</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>401</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string>Electrical Data</string>
    </property>
    <widget class="QLabel" name="BatteryImage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="BatteryPrecentageProgressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>381</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>78</number>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>710</y>
       <width>212</width>
       <height>18</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="WheelConnectionGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="WheelConnectionLbl">
        <property name="text">
         <string>Steering Wheel Connection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="WheelConnectionStatusImage">
        <property name="text">
         <string>LED</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>191</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pack Information</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="PrimaryVoltageGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="BusVoltageLbl">
          <property name="text">
           <string>Bus Voltage</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="BusVoltageSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="BusCurrentLbl">
          <property name="text">
           <string>Bus Current</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="BusCurrentSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="RinehartVoltageLbl">
          <property name="text">
           <string>RInehart Voltage</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="RinehartVoltageSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>381</width>
       <height>128</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Critical Data</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="6">
         <widget class="QDoubleSpinBox" name="CommandedTorqueSbx">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="DriveDirectionTbx">
          <property name="text">
           <string>Direction</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="commandedTorqueLbl">
          <property name="text">
           <string>Commanded Torque:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="DriveDirectionLbl">
          <property name="text">
           <string>Drive Direction: </string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLabel" name="InverterStatusLED">
          <property name="text">
           <string>LED</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="InverterStatusLbl">
          <property name="text">
           <string>Inverter Status: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLabel" name="IMDFaultLED">
          <property name="text">
           <string>LED</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLabel" name="BMSFaultLED">
          <property name="text">
           <string>LED</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="IMDFaultLbl">
          <property name="text">
           <string>IMD Fault: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="BMSFaultLbl">
          <property name="text">
           <string>BMS Fault:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="readyToDriveLbl">
          <property name="text">
           <string>Ready to Drive:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="readyToDriveLED">
          <property name="text">
           <string>LED</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="4">
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="prechargeStateLbl">
          <property name="text">
           <string>Precharge State</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="prechargeStateTbx">
          <property name="text">
           <string>State</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>381</width>
       <height>110</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Cell Infomation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="minCellVoltageLbl">
          <property name="text">
           <string>Min Cell Voltage</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="minCellVoltageSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="maxCellVoltageLbl">
          <property name="text">
           <string>Max Cell Voltage</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="maxCellVoltageSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="averageCellVoltageLbl">
          <property name="text">
           <string>Average Cell Voltage</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="averageCellVoltageSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="3">
         <widget class="QLabel" name="averageCellTempLbl">
          <property name="text">
           <string>Average Cell Voltage</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QDoubleSpinBox" name="averageCellTempSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QDoubleSpinBox" name="maxCellTempSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QDoubleSpinBox" name="minCellTempSbx">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="3">
         <widget class="QLabel" name="minCellTempLbl">
          <property name="text">
           <string>Min Cell Temp</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="3">
         <widget class="QLabel" name="maxCellTempLbl">
          <property name="text">
           <string>Max Cell Temp</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="CarImage">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>381</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string>Image of Car</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="FRWheelGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="FRWheelSpeedLbl">
       <property name="text">
        <string>Wheel Speed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="FRWheelSpeedSbx">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="FRWheelHeightLbl">
       <property name="text">
        <string>Suspension</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="FRWheelHeightSbx">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>670</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="BRWheelGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="BRWheelSpeedLbl">
       <property name="text">
        <string>Wheel Speed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="BRWheelSpeedSbx">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="BRWheelHeightLbl">
       <property name="text">
        <string>Suspension</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="BRWheelHeightSbx">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>670</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="BLWheelGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="BLWheelSpeedLbl">
       <property name="text">
        <string>Wheel Speed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="BLWheelSpeedSbx">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="BLWheelHeightLbl">
       <property name="text">
        <string>Suspension</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="BLWheelHeightSbx">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="FLWheelGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="FLWheelSpeedLbl">
       <property name="text">
        <string>Wheel Speed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="FLWheelSpeedSbx">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="FLWheelHeightLbl">
       <property name="text">
        <string>Suspension</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="FLWheelHeightSbx">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>11</y>
      <width>144</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="DriveModeGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="DriveModeLbl">
       <property name="text">
        <string>Drive Mode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="DriveModeTbx">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSelect_Serial_Port"/>
    <addaction name="Dark_Mode"/>
    <addaction name="About_Dlg"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Dark_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="About_Dlg">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSelect_Serial_Port">
   <property name="text">
    <string>Select Serial Port</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
